ux/discovery/discovery-overview.adoc

```diff
@@ -1 +1,437 @@
-= Discovery Assets
+= DISCOVERY:Work Item History
+:imagesdir: images
+
+== Overview
+Before we start the wireframe design of the work item history. We need to understand the issue and users’ scenario. The document is to research relevant aspects by asking  the following questions:
+
+. How the competitive product display history ?
+. What is the using scenario of history and activity ?
+. What is the differences between history and activity ?
+. Where to place the history function ?

```

Comment:
Please remove all of the spaces between the end of the text and the question mark.